airflow_prometheus_exporter/prometheus_exporter.py

```diff
@@ -372,6 +379,32 @@ def get_num_queued_tasks():
             .count()
         )
 
+def insert_latest_successful_run(session, insert_dict):
```

Comment:
since you are passing a dict, maybe rename this to `insert_latest_successful_runs`?